packages/route-viewer-overlay/src/RouteViewerOverlay.story.js

```diff
@@ -38,6 +40,20 @@ const Template = args => (
   </>
 );
 
+/** Should zoom to pattern. */
+
+const WithChangingRoute = () => {
+  const [rtData, setRouteData] = useState(routeData);
+
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
Could we possibly swap this to an interval that switches back and forth?

Reply:
Sure, updated in https://github.com/opentripplanner/otp-ui/pull/485/commits/fb1c25df7b28b974859562e58e13e0e468377953.